package.json

```
@@ -68,7 +68,7 @@
     "simple-git-hooks": "^2.9.0",
     "sync-request": "^6.1.0",
     "tsx": "^4.16.2",
-    "typescript": "5.5.2",
+    "typescript": "^5.6.2",
```

Let's keep the same specs across the workspace:
```suggestion
    "typescript": "5.6.2",
```